src/sources/host_metrics/cpu.rs

```diff
@@ -34,18 +34,19 @@ impl HostMetrics {
                                 self.counter(
                                     name,
                                     timestamp,
-                                    times.nice().get::<second>(),
```

Comment:
Did you mean to drop `nice` in this PR? I think the intent of the issue was just to _add_ iowait but leave the others.

Reply:
You're right, reverted.